Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt

```diff
@@ -0,0 +1,147 @@
+                      Analytical Graphics, Inc.
+  Software Grant and Corporate Contributor License Agreement ("Agreement")
+           http://www.agi.com/licenses/corporate-cla-agi-v1.0.txt
```

Comment:
Consider making a new CLA repo (or something similar) and using github pages.  Result could be something like:

https://analyticalgraphicsinc.github.io/CLA/individual-cla-agi-v1.0.txt